modules/mobile/app/services/mobile/v0/lighthouse_direct_deposit/configuration.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require Rails.root.join(*%w[lib lighthouse direct_deposit configuration])
+
+module Mobile
+  module V0
+    module LighthouseDirectDeposit
+      # Configuration for the Mobile::V0::LighthouseDirectDeposit::Service
+      #
+      class Configuration < ::DirectDeposit::Configuration
+        def service_name
+          'MOBILE_LIGHTHOUSE_DIRECT_DEPOSIT'
+        end
+
+        def mobile_settings
+          Settings.mobile_lighthouse
+        end
+
+        def token_service
+          url = "#{settings.host}/#{TOKEN_PATH}"
+          token = settings.access_token
+
+          @token_service ||= Auth::ClientCredentials::Service.new(
+            url, API_SCOPES, mobile_settings.client_id, token.aud_claim_url, mobile_settings.rsa_key
+          )
+        end
```

Comment:
We recently had a conversation with LH and benefits team about a new policy change that lets us share the client id of other vets-api services rather than using our own. We should be able to not create our own configuration and just use the teams that initially implemented the service.

Reply:
+1

Reply:
Oh right let me remove this